.appveyor.yml

```diff
@@ -1,14 +1,17 @@
 install:
+  - ps: Install-Product node 10
   - appveyor-retry appveyor DownloadFile https://win.rustup.rs/ -FileName rustup-init.exe
   - if not defined RUSTFLAGS rustup-init.exe -y --default-host x86_64-pc-windows-msvc --default-toolchain nightly
   - set PATH=%PATH%;C:\Users\appveyor\.cargo\bin
+  - set RUST_BACKTRACE=1
+  - rustup target add wasm32-unknown-unknown --toolchain nightly
```

Comment:
As discussed on IRC, this is because the commands use the noinstall mode, so that we only download binaries the one time and then copy the already downloaded binary to each fixture.